sim/shaman/elemental/dragonwrath.go

```diff
@@ -39,11 +47,29 @@ func customFulminationHandler(sim *core.Simulation, spell *core.Spell, result *c
 		copySpell = spell.Unit.RegisterSpell(copyConfig)
 	}
 
-	copySpell.ApplyEffects = fulminationFactory(totalDamage)
+	copySpell.ApplyEffects = damageFactory(totalDamage)
 	cata.CastDTRSpell(sim, copySpell, result.Target)
 }
 
-func fulminationFactory(damage float64) func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+func customEarthquakeHandler(sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+	copySpell := spell.Unit.GetSpell(spell.WithTag(71086))
+
+	shaman := spell.Unit.Env.Raid.GetPlayerFromUnit(spell.Unit).(shaman.ShamanAgent).GetShaman()
+	baseDamage := shaman.ClassSpellScaling*0.32400000095 + 0.11*spell.SpellPower()
+
+	if copySpell == nil {
+		copyConfig := cata.GetDRTSpellConfig(spell)
+		copyConfig.Cast = core.CastConfig{}
+		copyConfig.Flags |= core.SpellFlagIgnoreResists // Doesnt get double damage reduced by armor
+		copySpell = spell.Unit.RegisterSpell(copyConfig)
+	}
+
+	copySpell.ApplyEffects = damageFactory(baseDamage)
```

Comment:
This would cause EQ to only replicate damage to the target instead of all targets. See above, but I think the whole 'proc' is replicated? Or am I miss reading the log and EQ overlapped for some reason? Did not find many logs tbh

Reply:
If you're talking about my screenshot in the ele dtr issue, i should have mentionned that it was on 3 targets. Each damage can duplicate independently of each other. For a same "pulse" of EQ, you can have 0 1 2 3 dup. And it only does damage to the target it procs on

Reply:
Yeah I missed that one and looked out for an EQ log myself. Looking at your proc chance in your log at about 2.6%, thatwould somewhat align with the proc rate in the sim right now for AoE spells for shamans.

The 3x replication occurence would be 2.6%^3 however which is like 0.00176% or a one in 57k occurrence. So either lucky, or might there be a possiblity that the spell gets duped whole but from the effected target as the source? Not sure where you tested.

Reply:
that might be a thing yes, i was doing this 
![image](https://github.com/user-attachments/assets/6cf90e25-bbed-4fdb-9345-af93d3c18039)
that would also explain why in the other log, there is a new target out of nowhere